code/API_definitions/qos-profiles.yaml

```
@@ -405,7 +382,7 @@ components:
           - Support for predefined profiles QOS_S, QOS_M, QOS_L, and QOS_E
```

```suggestion
          - Support for predefined profiles Small (QOS_S), Medium (QOS_M), Large (QOS_L), and Extra large (QOS_E)
```

---

The names are coming from https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/API_documentation/QoSProfile_Mapping_Table.md and are only examples. And QoS_E in the table is actually not "extra large", but actually the opposite (derived from "Enhanced Communication" or "Enhanced Connectivity").

But as the table is not using any further terms, I wouldn't start to introduce them here. But adding a "like" as these actually only examples, there are currently no globally defined profile names.

```suggestion
          - Support for predefined profile names like `QOS_E`, `QOS_S`, `QOS_M`, and `QOS_L`
```

---

> The names are coming from https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/API_documentation/QoSProfile_Mapping_Table.md and are only examples. And QoS_E in the table is actually not "extra large", but actually the opposite (derived from "Enhanced Communication" or "Enhanced Connectivity").
> 
> But as the table is not using any further terms, I wouldn't start to introduce them here. But adding a "like" as these actually only examples, there are currently no globally defined profile names.

Should we remove https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/API_documentation/QoSProfile_Mapping_Table.md from the repo?